ipaserver/install/httpinstance.py

```diff
@@ -235,12 +236,13 @@ def set_mod_ssl_protocol(self):
                                    '+TLSv1 +TLSv1.1 +TLSv1.2', False)
 
     def set_mod_ssl_logdir(self):
-        installutils.set_directive(paths.HTTPD_SSL_CONF,
-                                   'ErrorLog',
-                                   'logs/error_log', False)
-        installutils.set_directive(paths.HTTPD_SSL_CONF,
-                                   'TransferLog',
-                                   'logs/access_log', False)
+        if NAME is not 'debian':
```

Comment:
Could you please handle this once again in the `ipaplatform` module? Perhaps extend the tasks with something like `setup_httpd_logging()`.